site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -9,7 +9,73 @@ docs: "DOCS-1419"
 
 This topic describes possible issues users might encounter when using NGINX Gateway Fabric. When possible, suggested workarounds are provided.
 
-### NGINX fails to reload
+### **General Troubleshooting**
+
+When attempting to diagnose a problem or get support, there are a few important data points that can be collected to help with understanding what issues may exist.
+
+#### Resource Status
+
+To get the status of a resource, use `kubectl describe`. For example, to check the status of the `coffee` HTTPRoute:
+
+```shell
+kubectl describe httproutes.gateway.networking.k8s.io coffee [-n namespace]
+```
+
+```text
+...
+Status:
```

Comment:
would it make sense to show an example of an error rather than a successful case?